Modix.Services/Starboard/StarboardHandler.cs

```diff
@@ -31,80 +34,76 @@ public Task Handle(ReactionRemoved notification, CancellationToken cancellationT
+                if (!_service.IsAboveReactionThreshold(message, emote))
                 {
                     await _service.RemoveFromStarboard(channel, message);
                 }
                 else
                 {
-                    await starboardMessage.ModifyAsync((messageProperties) =>
-                    {
-                        messageProperties.Embed = embed;
-                        messageProperties.Content = FormatContent(message, reaction);
-                    });
+                    await starboardMessage.ModifyAsync((messageProperties)
+                        => messageProperties.Content = FormatContent(message, emote));
                 }
             }
             else
             {
-                if (_service.IsAboveReactionThreshold(message, reaction))
+                if (_service.IsAboveReactionThreshold(message, emote))
                 {
-                    var embed = GetStarEmbed(channel, message);
-                    var quoteUrl = _service.BuildContextUrl(channel, message);
-
+                    var embed = GetStarEmbed(message);
                     await _designatedChannelService.SendToDesignatedChannelsAsync(
                         channel.Guild,
                         DesignatedChannelType.Starboard,
-                        FormatContent(message, reaction),
+                        FormatContent(message, emote),
                         embed);
                 }
             }
         }
 
-        public string FormatContent(IUserMessage message, IReaction reaction)
+        private string FormatContent(IUserMessage message, IEmote emote)
         {
-            var reactionCount = _service.GetReactionCount(message, reaction);
-            return string.Format(_contentString, reactionCount, GetStarEmote(reactionCount), message.Id);
+            var reactionCount = _service.GetReactionCount(message, emote);
+            return string.Format(_contentString, reactionCount, _service.GetStarEmote(reactionCount), message.Id);
         }
-
-        //TODO: Figure out where to put clickable link to original message
-        // Preferably in markdown format --> [Show More](link)
-        public Embed GetStarEmbed(IGuildChannel channel, IUserMessage message)
+        
+        private Embed GetStarEmbed(IUserMessage message)
         {
             var author = message.Author as IGuildUser;
-            var builder = new EmbedBuilder()
+            var builder = _quoteService.BuildQuoteEmbed(message, author)
                 .WithTimestamp(message.Timestamp)
                 .WithColor(new Color(255, 234, 174))
-                .WithAuthor(author.Nickname ?? author.Username, author.GetAvatarUrl())
-                .WithDescription(message.Content)
-                .WithFooter($"Posted in #{message.Channel.Name}");
-            //embed.Author.Url = _service.BuildContextUrl(channel, message);
+                .WithAuthor(
+                    author.Nickname ?? author.Username,
```

Comment:
Wondering if we should include the discriminator as well, like `$"{author.Nickname ?? author.Username}#{author.Discriminator}"`. Up to you.

If so, we really ought to add an extension method somewhere to get that string from an `IGuildUser`. Maybe in `Modix.Services\Utilities\GuildUserExtensions.cs`.
If not, there's an existing extension in the above file we can use in place of `author.Nickname ?? author.Username` I suppose.

Reply:
Personal (but unexperienced w/ Discord bots) opinion: don't bother. They can be anything. Having the user ID would be more valuable but it's incredibly long.